docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/970_special_report.content.yml

```diff
@@ -84,4 +85,142 @@
       field_override_card_title:
         - value: Alternate Title
       field_override_card_longer_descr:
-        - value: "Over the last 5 decades, mortality rates have declined for most cancers, largely due to improvements in prevention, detection, and treatment.  These advances have been fueled by breakthrough studies of complex environmental and genetic interactions that can lead to cancer, the identification of targets that drive cancer, and the development of effective interventions against those targets."              
\ No newline at end of file
+        - value: "Over the last 5 decades, mortality rates have declined for most cancers, largely due to improvements in prevention, detection, and treatment.  These advances have been fueled by breakthrough studies of complex environmental and genetic interactions that can lead to cancer, the identification of targets that drive cancer, and the development of effective interventions against those targets."
+      ######## End Borderless card ###########
+      ######## Begin Raw HTML Block ###########
+    - entity: 'paragraph'
+      type: "cgov_card_raw_html"
+      field_raw_html:
+        - format: "raw_html"
+          value: |
+            <div class="row flex-columns">
+              <div class="medium-10 columns align-center">
+                <div class="special-report-raw-html-container">
+                  <h2>
+                    <A href="https://www.cancer.gov/">Driving Discovery</a>
```

Comment:
can formatting be applied here? And what's up with `<A `

Reply:
Will update!